### PR TITLE
ENG-2268 #comment Update shareModal component add share to feed option

### PR DESCRIPTION
**Pull request by patricia-mps**

### CODE REVIEW

File: src/components/shareModal/ShareModal.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ const ShareModal: FC<Props> = ({
   openModal,
   onClickCopyToClipboard,
   stickToBottomMobile = true,
+  showShareToFeed,
+  onclickShareToFeed,
```
Comment (pedroguia) [inline src/components/shareModal/ShareModal.tsx:22]: onclickShareToFeed => onClickShareToFeed (capital C in click)

File: src/components/shareModal/ShareModal.tsx [hunk 2]
```diff
@@ -18,6 +18,8 @@ const ShareModal: FC<Props> = ({
   openModal,
   onClickCopyToClipboard,
   stickToBottomMobile = true,
+  showShareToFeed,
```
Comment (pedroguia) [inline src/components/shareModal/ShareModal.tsx:21]: Could have a default value, depending on where this component will be used the most:
- true if business
- false other projects

File: src/components/shareModal/ShareModal.types.ts [hunk 3]
```diff
@@ -14,6 +14,8 @@ export interface ModalBodyProps {
   showCopyToClipboard: boolean;
   windowLocationHref: string;
   onClickCopyToClipboard(showToast: boolean): void;
+  showShareToFeed: boolean;
```
Comment (pedroguia) [inline src/components/shareModal/ShareModal.types.ts:17]: Not required if you set the default value

**Review by pedroguia — CHANGES_REQUESTED** [review]

File: src/components/shareModal/ShareModal.tsx (hunk 1 above)
Comment (patricia-mps) [reply src/components/shareModal/ShareModal.tsx]: Done.

File: src/components/shareModal/ShareModal.types.ts (hunk 3 above)
Comment (patricia-mps) [reply src/components/shareModal/ShareModal.types.ts:17]: Done.

**Review by pedroguia — APPROVED** [review]

**Review by itsnunolemos — APPROVED** [review]

**Pull request merged**